CityOfDetroit/bloom
# Translate Filter and Eligibility buttons

## Issue

- Closes # (issue)
- Addresses # (issue)

## Description

Adds translations for the Filter and Eligibility buttons on the listings and home page.
![image](https://user-images.githubusercontent.com/82653098/130672938-69989c19-5d48-4fae-a388-aca434594d18.png)
![image](https://user-images.githubusercontent.com/82653098/130672966-60228d56-e21b-402a-abe8-164b9e46c013.png)
![image](https://user-images.githubusercontent.com/82653098/130672982-e215294c-4951-4a88-a631-1d78163b5f10.png)
![image](https://user-images.githubusercontent.com/82653098/130673026-951f4237-5916-4f7c-a4d5-ad65ae4b0d7e.png)


## Type of change


## How Can This Be Tested/Reviewed?

Change the language on the listings page and check the buttons adjust accordingly. 


## Checklist:
